### PR TITLE
Styled landing page, deleted navbar from landingpage

### PR DESCRIPTION
**Pull request by Jay-am**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -10,7 +10,9 @@
     <%#= stylesheet_pack_tag 'application', media: 'all' %> <!-- Uncomment if you import CSS in app/javascript/packs/application.js -->
   </head>
   <body>
-    <%= render 'shared/navbar' %>
+    <% if controller_name != 'pages' && action_name != 'home' %>
```
Comment (AbelvanEsseveldt) [inline app/views/layouts/application.html.erb]: Why did you change this?

Comment (Jay-am) [reply app/views/layouts/application.html.erb]: This will make sure that the navbar will be shown on every page except the landingpage

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+
+      <div class="home-page-categories">
+        <img src="/assets/coffee.jpg" alt="Coffee">
+        <img src="/assets/breakfast.png" alt="Breakfast">
+        <img src="/assets/breakfast.png" alt="Brunch">
+        <img src="/assets/dinner.jpg" alt="Lunch">
+        <img src="/assets/dinner.jpg" alt="Dinner">
+        <img src="/assets/drinks.png" alt="Drinks">
+        <img src="/assets/icecream.png" alt="Ice Cream">
+      </div>
```
Comment (kaori-kk) [inline app/views/pages/home.html.erb]: We need to implement filtering by types
